### PR TITLE
Transients manager

### LINKED ISSUE #45: Add a Transient Manager
Hello, I think we need a transient manager.
- Creation (based on `Timber::transient`?)
- Deletion (based on YAML config)

I'll push a PR soon about this one.

### PR DESCRIPTION
**Pull request by lusimeon**
This PR introduce a "Transients Manager" used to manage WordPress transients. This include :
- Run transients cleaner tool from [studiometa/wp-toolkit](https://github.com/studiometa/wp-toolkit/tree/master#transient-cleaner) if a YAML configuration is specified in `config/transients_cleaner.yml`

Closes #45.

### CODE REVIEW

File: template/web/wp-content/themes/<%= slug %>/app/Managers/AssetsManager.php [hunk 1]
```diff
@@ -18,6 +18,6 @@ class AssetsManager implements ManagerInterface {
 	 * @inheritDoc
 	 */
 	public function run() {
-		new Assets( get_template_directory() );
+		return new Assets( sprintf( '%s/config/', get_template_directory() ) );
```
Comment (titouanmathis) [inline template/web/wp-content/themes/<%= slug %>/app/Managers/AssetsManager.php]: **issue:** This change is not tied to the TransientManager feature, it should be made in a different PR

**Review by titouanmathis — CHANGES_REQUESTED** [review]
IMO, the move of the `assets.yaml` configuration should be made in another PR to discuss it separately from the `TransientsManager` feature.

File: template/web/wp-content/themes/<%= slug %>/app/Managers/AssetsManager.php (hunk 1 above)
Comment (lusimeon) [reply template/web/wp-content/themes/<%= slug %>/app/Managers/AssetsManager.php]: See #61

**Comment by titouanmathis** [comment]
This has been added to the [v1.0.0-alpha4](https://github.com/studiometa/wp-toolkit/releases/tag/1.0.0-alpha4) version of the [WP Toolkit](https://github.com/studiometa/wp-toolkit/), I am closing this PR.
